frontend/webapp/static/js/custom/display_info.js

```diff
@@ -28,11 +28,14 @@ mapHelpText_system['field_hijack_status'] += '<li>Ignored: the event was a false
 mapHelpText_system['field_hijack_status'] += '<li>Resolved: the event was a true positive that is now resolved.</li>';
 mapHelpText_system['field_hijack_status'] += '<li>Outdated: the event was triggered by a configuration that is now deprecated.</li></ul>';
 
-mapHelpText_system['field_hijack_type'] = 'The type of the hijack:<ul>';
-mapHelpText_system['field_hijack_type'] += '<li>S → Sub-prefix hijack</li>';
-mapHelpText_system['field_hijack_type'] += '<li>0 → Type-0 exact-prefix hijack</li>';
-mapHelpText_system['field_hijack_type'] += '<li>1 → Type-1 exact-prefix hijack</li>';
-mapHelpText_system['field_hijack_type'] += '<li>Q → Squatting attack </li></ul>';
+mapHelpText_system['field_hijack_type'] = 'The type of the hijack in 3 dimensions: prefix|path|data plane<ul>';
+mapHelpText_system['field_hijack_type'] += '<li>[Prefix] "S" → Sub-prefix hijack</li>';
```

Comment:
This is not an optimized way to create multiline strings in javascript. we need to change them all.. @pgigis

Reply:
Yes but this should be done in another issue, not here. Here it is simply auxiliary. If not critical, resolve the commend and please create a separate issue.